### ISSUE TITLE
js Web Worker does not send cookies

### ISSUE THREAD

**Issue by bezua** [DevExpress/testcafe#5058]
### What is your Test Scenario?
Simple test, basic clicking, however html page is generated dynamically, using web workers

### What is the Current behavior?
The application is using cookie based auth.
I'm able to login (main thread, after login I get auth cookie), however web worker processes running in background do not attach auth cookie in the request header.
As a result, all requests made with web workers are failed.
 
### What is the Expected behavior?
cookie is shared inside a test case with a web workers
 
### What is your web application and your TestCafe test code?
pure js application
TestCafe 1.8.4

Basically there is nothing to show, just take a look at request header initiated with a web worker - no cookies there 

```

```
</details>

<details>
<summary>Screenshots:</summary>

```

```
</details>

### Steps to Reproduce:

1. Usa a page using web workers (sorry, can't share mine)
2. The page gives you a cookie
3. Use dev tools or fidller to capture the traffic
4. Take a look at request headers made by a web worker, there are no cookies there
 
### Your Environment details:
 
* testcafe version:                   1.8.4
* node.js version:                    12.6.3
* command-line arguments:  testcafe chrome testname
* browser name and version: latest chrome also IE
* platform and version:          win10

**Comment by SergeyShurygin** [comment]
We've tried to reproduce this issue, but didn't succeed. 
I've created a sample test that works on my side.  Please, modify it in order to reproduce your issue.
[test.zip](https://github.com/DevExpress/testcafe/files/4600190/test.zip)

Note, you are not discovering cookies in traffic in the devtools or the fiddler because the testcafe keeps cookies on its own proxy server and synchronizes them with the client.  You need to analyze the traffic on the server side if you want to check them.

**Comment by vandalton** [comment]
I work in the same company as @bezua and I used to work on the same project a year ago or so. I modified your project to narrow down the problem, because it affects also my current team. Sorry it took so long.

When we instantiate a worker in the normal fasion and worker's code makes a request, cookies are correctly added to the request. However, in our code we use `URL.createObjectURL` and `importScripts` functions. That particular combination causes the problem. When I just run the browser, cookies are present in all request made by the worker and in the request for the worker's source itself. When testcafe runs the test, cookies are not present in these requests.

Here is the short movie I created to describe the problem and project modifications: https://youtu.be/lqS2Xfa6NSo

[test_modified.zip](https://github.com/DevExpress/testcafe/files/4722768/test_modified.zip)
